### DISCUSSION TITLE
Possible issue with admin auth endpoint in v23

### DISCUSSION THREAD

**Discussion by jjones3535a** [pocketbase/pocketbase#5822] [Q&A]
I'm having an issue with the new RC v23 and accessing await pb.admins.authWithPassword() - i kept getting this error: "The requested resource wasn't found", but with v22 await pb.admins.authWithPassword() works fine.  

As a quick test if I visit a PB v0.22.23 url like: https://pb1.example.com/api/admins/auth-with-password, i get this:  {"code":401,"message":"The request requires valid admin authorization token to be set.","data":{}}
which is to be expected.

However if i visit the v23 instance (in this case RC8) i get this in the browser: {"data":{},"message":"File not found.","status":404}

Is anyone having issues with v23 and this endpoint "/api/admins/auth-with-password" on being not found/available?

**Comment by ganigeorgiev** [comment]
It seems that you are using an old JS/Dart SDK. PocketBase v0.23.0-rc+ is compatible with JS SDK v0.22.0-rc+ and Dart SDK v0.19.0-rc+.

With v0.23.0-rc admins are converted to system `_superusers` auth collection records, aka. there is longer `/api/admins` endpoint.

To fix your error, upgrade your JS/Dart SDK version to the compatible ones and replace `pb.admins.*` with `pb.collection("_superusers").*`.

For all changes please refer to the [v0.23.0 CHANGELOG](https://github.com/pocketbase/pocketbase/blob/develop/CHANGELOG.md) (starting from the bottom; make sure to check also the related SDKs changelogs).

**Reply by jjones3535a** [reply]
Thank you.
